index.html

```diff
@@ -11,6 +11,9 @@
       href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200&family=Poppins:ital,wght@0,700;1,200&family=Roboto+Slab&display=swap"
       rel="stylesheet"
     />
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+   
   </head>
   <body>
```

Comment:
- [ ] [OPTIONAL] Kindly add padding of 0 and a margin of 0 on the `body` tag to remove them on some browsers